include/rocksdb/perf_level.h

```diff
@@ -13,8 +13,8 @@ namespace rocksdb {
 
 // How much perf stats to collect. Affects perf_context and iostats_context.
 
-enum PerfLevel : char {
-  kUninitialized = -1,            // unknown setting
+enum PerfLevel : unsigned char {
```

Comment:
I'll suggest someone else carry the torch as my PR failed the travisci gate and I'm not interested in signing the CLA...